.make/.minishift/fabric8-toggles-config.yml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+items:
+- apiVersion: v1
+  data:
+    fabric8.toggles.url: http://fabric8-toggles-f8toggles.192.168.64.4.xip.io/api
```

Comment:
this is not going to work for everyone. We might consider using an env var for this, instead.